SmithyDafnyMakefile.mk

```diff
@@ -56,6 +56,15 @@ SMITHY_MODEL_ROOT := $(LIBRARY_ROOT)/Model
 CODEGEN_CLI_ROOT := $(SMITHY_DAFNY_ROOT)/codegen/smithy-dafny-codegen-cli
 GRADLEW := $(SMITHY_DAFNY_ROOT)/codegen/gradlew
 
+include $(SMITHY_DAFNY_ROOT)/SmithyDafnySedMakefile.mk
+
+# This flag enables pre-processing on extern module names.
+# This pre-processing is required to compile to Python and Go.
+# This is disabled by default.
+# This should be enabled in each individual project's Makefile if that project compiles to Python or Go.
+# This can be enabled by setting this variable to any nonempty value (ex. true, 1)
+ENABLE_EXTERN_PROCESSING?=
+
 ########################## Dafny targets
 
 # TODO: This target will not work for projects that use `replaceable` 
```

Comment:
As discussed offline, mind removing this comment since we've determined it's stale?